lib/patch-series.ts

```diff
@@ -198,6 +205,9 @@ export class PatchSeries {
         const publishToRemote: string | undefined = undefined;
 
         const project = await ProjectOptions.get(workDir, headCommit, cc, basedOn, publishToRemote, baseCommit);
+        if (typeof rangeDiff !== "undefined") {
```

Comment:
Can't we just use `if (rangeDiff)` for simplicity?

Reply:
No, that was an error on my part during renames (thanks for catching it).  It is supposed to be a check on `doRangeDiff`, which may not be defined.  Since it is a boolean, existence needs to be checked.

Reply:
> Since it is a boolean, existence needs to be checked.

Except now it is a string!